speed-dreams/source-2.2.3/data/data/indicators/config.xml

```diff
@@ -42,6 +44,7 @@
             </section>
             <section name="sound">
                 <attstr name ="source" val="warning_right_driftingleft.wav"/>
+                <attstr name="looping" val="false"/>
```

Comment:
You use 'false' as value here, but when you read in the XML you check for 'yes' and 'no'

Reply:
Should we use yes or no? I used it because the rest of speed dreams used it but this isn't the first time I messed up and I've already wasted 15 minutes on trying to find out why my things weren't enabled

Reply:
'yes' and 'no' is used in the menus for checkbox as far as I know, since you are not using those predefined macro's anyway, you can just use 'true' and 'false' and make macro's for those

Reply:
Basically every boolean value uses it, also cars for example.